CODIGO/BabelUI.bas

```diff
@@ -956,45 +956,31 @@ On Error GoTo HandleUpdateIntSetting_Err
             Call SaveSetting("OPCIONES", "UseExperimentalUI", Value)
         Case eEnableMusic
             If Value = 0 Then
-                Sound.Music_Stop
-                Musica = CONST_DESHABILITADA
+                ao20audio.stopallplayback
+                ao20audio.MusicEnabled = 0
             Else
-                Musica = CONST_MP3
-                Sound.NextMusic = MapDat.music_numberHi
-                Sound.Fading = 100
+                ao20audio.MusicEnabled = 0
             End If
         Case eEnableFx
-            Fx = Value
+            ao20audio.FxEnabled = Value
             If Value = 0 Then
-                Call Sound.Sound_Stop_All
+                Call ao20audio.stopallplayback
             End If
         Case eEnableAmbient
             If Value = 0 Then
-                AmbientalActivated = 0
-                Sound.LastAmbienteActual = 0
-                Sound.AmbienteActual = 0
-                Sound.Ambient_Stop
-            Else
-                AmbientalActivated = 1
-                Call AmbientarAudio(UserMap)
-
+                    'todo
```

Comment:
we need to support this setting

Reply:
No longer needed like much of the old code. 

Basically ambient is just a looping wav.

We play and stop wavs with the new interface
```
Call ao20audio.stopwav(SND_LLUVIAIN)
Call ao20audio.stopwav(SND_LLUVIAIN)
```

Reply:
Resolved in next patch